### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by Nate152**
# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Test A
- [ ] Test B

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: front-end/lambda-in/src/Components/Landing Page/Image Container/image-container.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './image-container.css';
+import Button from '@material-ui/core/Button';
+
+class ImageContainer extends React.Component {
```
Comment (jesuarva) [inline front-end/lambda-in/src/Components/Landing Page/Image Container/image-container.js]: I would suggest to name the file in CamelCase, so the file will match the Class name.
```
ImageContainer.js
```

File: front-end/lambda-in/src/Components/Landing Page/Navbar/navbar.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+//import { Navbar, NavItem, NavDropdown, Nav, MenuItem } from 'react-bootstrap';
+import './navbar.css';
+import Button from '@material-ui/core/Button';
+import Paper from '@material-ui/core/Paper';
+
+class NavBar extends React.Component {
```
Comment (jesuarva) [inline front-end/lambda-in/src/Components/Landing Page/Navbar/navbar.js]: I would suggest naming the file in CamelCase, so the file will match the Class name.
```
NavBar.js
```

File: front-end/lambda-in/src/Components/Landing Page/landing-page.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import NavBar from './Navbar/navbar.js';
+import ImageContainer from './Image Container/image-container.js'
+
+class LandingPage extends React.Component {
```
Comment (jesuarva) [inline front-end/lambda-in/src/Components/Landing Page/landing-page.js]: I would suggest naming the file in CamelCase, so the file will match the Class name.
```
LandingPage.js
```

**Comment by shobanavv** [comment]
There is a merge conflict. Git pull from master may show conflicts in your local branch. Modify it and if you could do one more push, we will be able to merge it.

**Pull request merged**